frontend/src/theming/index.less

```diff
@@ -9,3 +10,4 @@ body {
   -moz-osx-font-smoothing: grayscale;
   transition: background-color 0.5s, color 0.5s;
 }
+
```

Comment:
This could be the place in which adding the generalized color classes, given you think it makes sense.

Reply:
i agree, i'm adding here generalized color classes using global variables of antd (e.g. @primary-color)

Reply:
solved [Here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/a3577d187e8c33dca24fd65ab28bf43416e53455)